cli/src/main.rs

```diff
@@ -226,13 +232,17 @@ fn run() -> HolochainResult<()> {
         }
         .map_err(HolochainError::Default)?,
 
-        Cli::KeyGen { silent, path } => {
-            let passphrase = if silent {
+        Cli::KeyGen {
+            path,
+            quiet,
+            nullpass,
+        } => {
```

Comment:
```suggestion
        } => {
        // a default passphrase is used as the null passphrase, because
        // separately in the Conductor it will attempt to decrypt using the
        // default passphrase first, and then fallback on asking for user input
```